### PR TITLE
CAPA exception bug and update to latest version

### PR DESCRIPTION
**Pull request by phate1**
Hi

The CAPA module seemed to have been left unloved so I've had a go at bringing it up to date to work with the latest version and should also fix the bug I mentioned in #68 

Pete

### CODE REVIEW

File: processing/flare_capa/flare_capa.py [hunk 1]
```diff
@@ -24,30 +25,70 @@ class FlareCapa(ProcessingModule):
 
     def initialize(self):
         if not HAVE_CAPA:
-            raise ModuleInitializationError(self, 'Missing dependency: flare-capa')
+            raise ModuleInitializationError('Missing dependency: flare-capa')
```
Comment (Augustin-FL) [inline processing/flare_capa/flare_capa.py:28]:
```suggestion
            raise ModuleInitializationError(self, 'Missing dependency: flare-capa')
```

As discussed in https://github.com/certsocietegenerale/fame_modules/issues/68 , `ModuleInitializationError()` take two arguments

**Review by Augustin-FL — COMMENTED** [review]
The PR seems good, thanks ! 
It indeed fixes the capa module. Could you please fix the small issue with `ModuleInitializationError()` ?

**Comment by Augustin-FL** [comment]
Hi,

I will merge this PR and fix the `ModuleInitializationError` issue afterwards

Thanks again !

**Comment by phate1** [comment]
Thanks @Augustin-FL 
Sorry I didn't get to the fix I'm away from my pc this week

Pete

**Pull request merged**